src/callbacks_step/analysis.jl

```diff
@@ -444,14 +444,21 @@ function print_amr_information(callbacks, mesh::P4estMesh, solver, cache)
     elements_per_level .+= tree.quadrants_per_level
   end
 
-  min_level = findfirst(i -> i > 0, elements_per_level) - 1
-  max_level = findlast(i -> i > 0, elements_per_level) - 1
+  min_level_1 = findfirst(i -> i > 0, elements_per_level)
+  max_level_1 = findlast(i -> i > 0, elements_per_level)
 
-  for level = max_level:-1:min_level+1
-    mpi_println(" ├── level $level:    " * @sprintf("% 14d", elements_per_level[level + 1]))
-  end
-  mpi_println(" └── level $min_level:    " * @sprintf("% 14d", elements_per_level[min_level + 1]))
+  # Check if there is at least one level with an element
```

Comment:
If there is no level with an element, does that mean that the current rank does not have any elements?

Reply:
I don't know exactly, but in the example there are 24 trees and 96 elements and it fails when more than 24 MPI ranks (i.e. the number of trees, not elements?) are used. So in theory there are enough elements such that every rank can have at least one.

Reply:
Yes, that's our interpretation. @JoshuaLampert got this error when running a 2D P4est elixir with 24 trees in the mesh. Everything was fine up to 25 MPI ranks but it threw errors with 25 ranks or more.

Reply:
I don't remember exactly how everything was set up. We should be able to distribute the cells, not only the trees, @sloede? But this shows that there might be some problems with that.

Reply:
I think so too - parallelization is not over trees but over cells. @lchristm have you encountered something like this before? And did you know that - apparently - we can now run simulations with some domains having zero cells (which is really important for certain multi-physics applications)?

Reply:
I can confirm that the cells are distributed (i.e. different parts of a tree can live on separate ranks). There are some known problems with empty ranks (see #1096), but they are related to OrdinaryDiffEq if I recall correctly.

Reply:
OK. I think it's ok to close this here now, but could you please create an issue for the question why it seems like only trees but not cells get distributed, @JoshuaLampert? this seems like a bug to me...

Reply:
Done in #1329.